pkg/kubelet/configmap/configmap_manager_test.go

```diff
@@ -170,3 +170,33 @@ func TestCacheBasedConfigMapManager(t *testing.T) {
 		}
 	}
 }
+
+func TestGetConfigMap(t *testing.T) {
+	fakeClient := &fake.Clientset{}
+	store := manager.NewObjectStore(getConfigMap(fakeClient), clock.RealClock{}, noObjectTTL, 0)
+	cmm := &configMapManager{
+		manager: manager.NewCacheBasedManager(store, getConfigMapNames),
+	}
+	mycm := configMapsToAttach{
+		volumes: []string{"myvol"},
+	}
+
+	cmm.RegisterPod(podWithConfigMaps("myns", "mypod", mycm))
+	if _, err := cmm.GetConfigMap("myns", "myvol"); err != nil {
+		t.Fatalf("error getting configmap: %v", err)
+	}
+
+	mycm2 := configMapsToAttach{
+		volumes: []string{"myvol2"},
+	}
+
+	cmm.RegisterPod(podWithConfigMaps("myns", "mypodname", mycm2))
+
+	c, err := cmm.GetConfigMap("myns", "myvol2notexisting")
+	if err == nil {
+		t.Fatalf("expected error for not existing config map has not been thrown")
+	}
+	if c != nil {
```

Comment:
nit: checking this seems redundant with the `err == nil` check above.  in which case line 195 and 196 can be combined like 185.

Reply:
Thank for your review @sjenning will change that!